test_runner/performance/README.md

```diff
@@ -1,3 +1,19 @@
+# Running locally
+
+First make a release build. The profiling flag is optional, used only for tests that
+generate flamge graphs. The `-s` flag just silences a lot of output, and makes it
+easier to see if you have compile errors without scrolling up.
+`BUILD_TYPE=release CARGO_BUILD_FLAGS="--features=testing,profiling" make -s -j8`
```

Comment:
This doesn't work in my case. Even `cargo build --features profiling` with the following error:
https://pastebin.com/d1GWtSKx
Haven't investigated that, will try in the morning.

Reply:
Looks like the libc crate exports different types on linux (`timer_t`) vs osx (`time_t`), and we use `timer_t` in pprof https://github.com/neondatabase/pprof-rs/blob/wallclock-profiling/src/timer.rs#L24

Reply:
I added a note explaining this.